VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.ts

```diff
@@ -28,6 +28,14 @@ export class DeviceTypeService {
         );
     }
 
+    isIOS(): boolean {
+        return this.deviceDetectorService.os.toLowerCase() === 'ios';
```

Comment:
Could use the OS const from node_modules\ngx-device-detector\lib\device-detector.constants.d.ts